saleor/extensions/plugins/avatax/plugin.py

```diff
@@ -418,6 +420,23 @@ def taxes_are_enabled(self, previous_value: bool) -> bool:
             return previous_value
         return True
 
+    @classmethod
+    def validate_plugin_configuration(cls, plugin_configuration: "PluginConfiguration"):
+        """Validate if provided configuration is correct."""
+        missing_fields = []
+        configuration = plugin_configuration.configuration
+        configuration = {item["name"]: item["value"] for item in configuration}
+        if not configuration["Username or account"]:
+            missing_fields.append("Username or account")
+        if not configuration["Password or license"]:
+            missing_fields.append("Password or license")
+
+        if plugin_configuration.active and missing_fields:
+            error_msg = (
+                "To enable a plugin you need to provide missing value for the fields: "
+            )
+            raise ValidationError({"active": error_msg + ", ".join(missing_fields)})
```

Comment:
I would raise this error as a general one, without assigning it to the `active` field. It's related to a few fields usually.